pkg/app/mutate_app_psp_removal.go

```diff
@@ -37,6 +37,11 @@ const (
   podSecurityStandards:
     enforced: true`
 	topPriority = 150
+	// pspLabel values have to match the ones defined in pss-operator.
+	// See https://github.com/giantswarm/pss-operator/blob/main/service/controller/handler/pssversion/create.go#L25
+	// pspLabelKey has been escaped ('/' replaced with '~1') to fit JSONPatch format.
+	pspLabelKey = "team-shield.giantswarm.io~1psp"
```

Comment:
Non-blocking suggestion. We have a `policy` group we're using for related stuff, we could just tuck this under that as well.
```suggestion
	pspLabelKey = "policy.giantswarm.io~1psp-status"
```